engine/rpcserver_test.go

```diff
@@ -1688,3 +1688,64 @@ func TestGetManagedOrders(t *testing.T) {
 		t.Errorf("unexpected order result: %v", oo)
 	}
 }
+
+func TestRPCServer_unixTimestamp(t *testing.T) {
+	s := RPCServer{
+		Engine: &Engine{
+			Config: &config.Config{
+				RemoteControl: config.RemoteControlConfig{
+					GRPC: config.GRPCConfig{
+						TimeInNanoSeconds: false,
+					},
+				},
+			},
+		},
+	}
+	const sec = 1618888141
+	const nsec = 2
+	x := time.Unix(sec, nsec)
+
+	timestampSeconds := s.unixTimestamp(x)
+	if timestampSeconds != sec {
+		t.Errorf("have %d, want %d", timestampSeconds, sec)
+	}
+
+	s.Config.RemoteControl.GRPC.TimeInNanoSeconds = true
+	timestampNanos := s.unixTimestamp(x)
+	if want := int64(sec*1_000_000_000 + nsec); timestampNanos != want {
+		t.Errorf("have %d, want %d", timestampSeconds, want)
+	}
+}
+
+func TestRPCServer_GetTicker_LastUpdatedNanos(t *testing.T) {
+	server := RPCServer{Engine: RPCTestSetup(t)}
+
+	request := &gctrpc.GetTickerRequest{
+		Exchange: testExchange,
+		Pair: &gctrpc.CurrencyPair{
+			Delimiter: currency.DashDelimiter,
+			Base:      currency.BTC.String(),
+			Quote:     currency.USD.String(),
+		},
+		AssetType: asset.Spot.String(),
+	}
+
+	one, err := server.GetTicker(context.Background(), request)
+	if err != nil {
+		t.Error(err)
+	}
+
+	server.Config.RemoteControl.GRPC.TimeInNanoSeconds = true
+	two, err := server.GetTicker(context.Background(), request)
+	if err != nil {
+		t.Error(err)
+	}
+
+	now := time.Now().Unix()
+	if x := now - one.LastUpdated; x > 10 {
+		t.Errorf("have %d, want %d +- 10s", one.LastUpdated, now)
```

Comment:
To be honest I'm not really sure how exactly I can verify in a reliable way whether a timestamp is in nanoseconds or seconds. I increased the time tolerance to 5 minutes, would that be enough?

Reply:
Since the response from the endpoint is an int64, you can check if the size of it is in line with unix and unix nano sizes. A poorly formatted example:
```
	if one.LastUpdated > 9999999999 {
		t.Error()
	}
	if two.LastUpdated <= 9999999999 {
		t.Error()
	}
```

Reply:
That's really imprecise. If you insist, I'll do it exactly like that, but imho giving a time window is better. My latest commit made that time window 5 minutes, which should be absolutely enough and also gives a fairly proper test accuracy.

Reply:
@gloriousCode ?

Reply:
Sorry for the confusion, but what I was trying to highlight is that testing the time against a window like that isn't really testing much. The closeness of a returned ticker time versus the current time is not an RPCServer level concern and if there were an issue, that would be an exchange level test/fix. I was trying to say that since the time is not relevant, that you only really need to check whether the response is unix or unix nano. However @shazbert 's suggestion is better. You are comparing that the `time.Time` is what you expect and that it is a valid `time.Time`. So I think you should go with that instead.